backend/.env

```diff
@@ -0,0 +1 @@
+MONGODB_TEST_URI='mongodb://localhost:27017/testdb'
```

Comment:
I think .env files should be in .gitignore?

Reply:
Yes will add. Although I wanted to have everyone be able to pull the file at first since its being referenced in `server.js`